services/sources/publisher_backlist_service.py

```diff
@@ -0,0 +1,65 @@
+from datetime import datetime, timedelta, timezone
+import os
+import requests
+import json
+from typing import Optional
+
+from logger import create_log
+from mappings.UofM import UofMMapping
+from .source_service import SourceService
+
+
+logger = create_log(__name__)
+
+
+class PublisherBacklistService(SourceService):
+    def __init__(self):
+        self.airtable_auth_token = os.environ.get('AIRTABLE_KEY', None)
+
+        pass
+
+    def get_records(
+        self,
+        full_import: bool=False, # get all records from Airtable
+        start_timestamp: datetime=None,
+        offset: Optional[int]=None, # start at specific position in paginated results 
+        limit: Optional[int]=None
```

Comment:
I think we can remove the comments here!

Reply:
I agree